src/displayapp/screens/WatchFacePineTimeStyle.cpp

```diff
@@ -34,6 +34,7 @@
 #include "components/motion/MotionController.h"
 #include "components/settings/Settings.h"
 #include "displayapp/DisplayApp.h"
+#include "components/alarm/AlarmController.h"
```

Comment:
please move the new include up to the other component includes, to have them alphabetically ordered (just before the components/ble include please

Reply:
Done :^)